ui/src/views/storage/CreateVolume.vue

```diff
@@ -150,6 +150,9 @@ export default {
     }
   },
   computed: {
+    createVolumeFromVM () {
```

Comment:
minor - `createVolumeForVM` suits better?

Reply:
@shwstppr I think the intention (if you see related code) to say you're creating volume from the VM tab/resource view; we've similar createVolumeFromSnapshot (to create volume from the snapshot view)